doc/pages/keys.asciidoc

```diff
@@ -640,8 +645,9 @@ the wanted object:
 == Prompt commands
 
 When pressing `:` in normal mode, Kakoune will open a prompt to enter a command.
-The following keys are recognized by this mode to help edition.
-(See <<commands#,`:doc commands`>>)
+On validation the command line is stored in the *:* register (See <<registers#,`:doc registers`>>).
```

Comment:
I'm not sure what "validation" means here. Is it something to do with the way the `:` turns red if your command-line has a syntax error?

If it's just "when you press `<ret>`", then we can just say that.